crawl-ref/source/l-you.cc

```diff
@@ -69,6 +69,11 @@ LUARET1(you_race, string, species_name(you.species).c_str())
  * @treturn string
  * @function class
  */
+LUARET1(you_hand, string, species_hand_name(you.species).c_str())
+/*** Get noun for player's hands.
+ * @treturn string
+ * @function hand
+ */
```

Comment:
```suggestion
/*** Get noun for player's hands.
 * @treturn string
 * @function hand
 */
LUARET1(you_hand, string, species_hand_name(you.species).c_str())
```

ldoc comments appear before the function being documented, not after. Github won't let me suggest it, but you also need to make sure the doc comment for you_class remains in the right place.

Reply:
Fixed. I think the text highlighting in my editor confused me so I didn't realize the proper ordering. Thanks.